### PR TITLE
404 page nits given by @jankeromnes

### PR DESCRIPTION
**Pull request by arshdkhn1**

### CODE REVIEW

File: static/css/janitor-new.css [hunk 1]
```diff
@@ -82,7 +82,7 @@ main {
   overflow: auto;
   flex-grow: 1;
   /* viewport height - header height - 15px extra height(so that footer is visible) */
-  min-height: calc(100vh - 95px);
+  min-height: calc (100vh - 95px);
```
Comment (nt1m) [inline static/css/janitor-new.css]: I don't like this style personally.

File: app.js [hunk 2]
```diff
@@ -455,6 +455,13 @@ boot.executeInParallel([
     routes.notFoundPageNew(query.res, user);
   });
 
+  // New 404 Not Found page
+  app.route(/^\/404-new\/?$/, (data, match, end, query) => {
```
Comment (nt1m) [inline app.js]: Shouldn't this be replacing the block above ?

File: static/css/janitor-new.css (hunk 1 above)
Comment (jankeromnes) [reply static/css/janitor-new.css]: My bad, I meant the `(` in the comment, not the one after `calc`. I'll fix up this pull request.

File: app.js (hunk 2 above)
Comment (jankeromnes) [reply app.js]: Will fix this as well.

**Pull request merged**